test/resources/mnist/model_gpu/code/mnist.py

```diff
@@ -54,4 +54,6 @@ def model_fn(model_dir):
     model = torch.nn.DataParallel(Net())
     with open(os.path.join(model_dir, 'torch_model.pth'), 'rb') as f:
         model.load_state_dict(torch.load(f))
+    device = torch.device("cuda") if torch.cuda.is_available() else torch.device("cpu")
```

Comment:
Since this is gpu test. Don't we want to fail if cuda is not available?

Reply:
Updated in subsequent commit. Added condition to raise error if GPU is unavailable.